UnoCheck/Solutions/LinuxNinjaOpenUrlSolution.cs

```diff
@@ -0,0 +1,26 @@
+﻿using DotNetCheck.Models;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Solutions
+{
+	public class LinuxNinjaOpenUrlSolution : Solution
+	{
+		public override async Task Implement(SharedState sharedState, CancellationToken cancellationToken)
+		{
+			await base.Implement(sharedState, cancellationToken);
+
+			var ps = new ProcessStartInfo(LinuxNinjaOrgUrl)
+			{
+				UseShellExecute = true,
+				Verb = "open"
+			};
+			_ = Process.Start(ps);
+
+			ReportStatus("Please check the https://ninja-build.org/.");
```

Comment:
```suggestion
			ReportStatus("To install ninja, please visit https://ninja-build.org");
```

Reply:
Done!

Reply:
Make sure to push the commit that has the change before resolving the comment, thanks!

Reply:
Okay, I will!